docs/mxdoc.py

```diff
@@ -80,9 +80,9 @@ def build_r_docs(app):
 
 def build_scala_docs(app):
     """build scala doc and then move the outdir"""
-    scala_path = app.builder.srcdir + '/../scala-package/core/src/main/scala/ml/dmlc/mxnet'
+    scala_path = app.builder.srcdir + '/../scala-package'
     # scaldoc fails on some apis, so exit 0 to pass the check
-    _run_cmd('cd ' + scala_path + '; scaladoc `find . | grep .*scala`; exit 0')
+    _run_cmd('cd ' + scala_path + '; scaladoc `find . -type f -name "*.scala" | egrep \"\/core|\/infer\" | egrep -v \"Suite\"`; exit 0')
```

Comment:
you own this so I’m no adamant about it, but just wondering, will it always appear to succeed with the semicolons and ‘exit 0’? would it not be better to use && between the commands? I’m not familiar with _run_cmd() call.

Reply:
Yes, I have tested locally. This not how configuration should be dealt- hardcoding(I think that needs to be a separate task). I tweaked the existing code to add new classes that we worked on.